src/main/java/edu/umich/lib/dor/replicaexperiment/service/OcflFilesystemRepositoryClient.java

```diff
@@ -34,20 +32,17 @@ public OcflFilesystemRepositoryClient(Path rootPath, Path workPath) {
             .build();
     }
 
-    private VersionInfo createNewVersion(Curator curator, String message) {
-        return new VersionInfo().setUser(curator.username(), curator.email()).setMessage(message);
+    public OcflFilesystemRepositoryClient(OcflRepository repository) {
+        this.repo = repository;
```

Comment:
To enable mocking of the core object, in other languages, I've used a static factory method and had something like this for the core constructor. But this seems fine because Java supports multiple constructors, right?